jdl/models/jdl-enum.js

```diff
@@ -39,6 +39,16 @@ class JDLEnum {
     return stringifyValues(this.values).join(',');
   }
 
+  getValueJavadocs() {
```

Comment:
It didn't bother me the first time, but as I'm re-reading it again, I find it weird that we use "javadocs" instead of javadoc. WDYT?

Reply:
That is because this method returns multiple javadocs for all values of the ENUM.  
For example:-
`{
        "INDIA": "Country 1",
        "FRANCE": "Country 2"
        "ENGLAND": "Country 3",
      }`
Hence the plural

Reply:
oh right! thanks